src/webui/api/appcontroller.cpp

```diff
@@ -281,6 +281,10 @@ void AppController::preferencesAction()
     data["enable_os_cache"] = session->useOSCache();
     // Coalesce reads & writes
     data["enable_coalesce_read_write"] = session->isCoalesceReadWriteEnabled();
+#if (LIBTORRENT_VERSION_NUM >= 10202)
```

Comment:
It's really bad idea to have conditionally included fields in WebAPI result. You should either have some noop "caps" here in case of unsupported libtorrent version or provide it under different API version (IMO, bad choice).

Reply:
> You should either have some noop "caps" here in case of unsupported libtorrent version

I don't understand what this means. Can you clarify, please?

Reply:
I mean it should exist unconditionally but do nothing in case of libtorrent version that unsupports it.

Another way to make API stable is provide no options like this one until it is supported unconditionally.

Reply:
Ok, the WebAPI now always returns `false` if the libtorrent version does not support the option.